docs/cloud/visualize/interact-new-charts.md

```diff
@@ -144,105 +148,154 @@ The following aggregate functions are available for each selected dimension:
+This menu also provides the contribution of each node to the volume of the chart, and a break down of the anomaly rate of the queried data per node.
 
-Click on **X Instances** to display a dropdown of instances and nodes contributing to that composite chart. Each line in
-the dropdown displays an instance name and the associated node's hostname.
-
-### Nodes
-
-Click on **X Nodes** to display a dropdown of nodes contributing to that composite chart. Each line displays a hostname
-to help you identify which nodes contribute to a chart. You can also use this component to filter nodes directly on the
-chart.
+<img src="https://user-images.githubusercontent.com/70198089/236137765-b57d5443-3d4b-42f4-9e3d-db1eb606626f.png" width="900"/>
 
 If one or more nodes can't contribute to a given chart, the definition bar shows a warning symbol plus the number of
 affected nodes, then lists them in the dropdown along with the associated error. Nodes might return errors because of
 networking issues, a stopped `netdata` service, or because that node does not have any metrics for that context.
 
+### Instances dropdown
+
+In this dropdown, you can view or filter the instances contributing time-series metrics to the chart.  
+This menu also provides the contribution of each instance to the volume of the chart, and a break down of the anomaly rate of the queried data per instance.
+
+<img src="https://user-images.githubusercontent.com/70198089/236138302-4dd4072e-3a0d-43bb-a9d8-4dde79c65e92.png" width="900"/>
+
+### Dimensions dropdown
+
+In this dropdown, you can view or filter the original dimensions contributing time-series metrics to the chart.  
+This menu also presents the contribution of each original dimensions on the chart, and a break down of the anomaly rate of the data per dimension.
+
+<img src="https://user-images.githubusercontent.com/70198089/236138796-08dc6ac6-9a50-4913-a46d-d9bbcedd48f6.png" width="900"/>
+
+
+### Labels dropdown
+
+In this dropdown, you can view or filter the contributing time-series labels of the chart.  
+This menu also presents the contribution of each label on the chart,and a break down of the anomaly rate of the data per label.
+
+<img src="https://user-images.githubusercontent.com/70198089/236139027-8a51a958-2074-4675-a41b-efff30d8f51a.png" width="900"/>
+
 ### Aggregate functions over time
 
 When the granularity of the data collected is higher than the plotted points on the chart an aggregation function over
-time
-is applied. By default the aggregation applied is _average_ but the user can choose different options from the
-following:
-
-* Min
-* Max
-* Average
-* Sum
-* Incremental sum (Delta)
-* Standard deviation
-* Median
-* Single exponential smoothing
-* Double exponential smoothing
-* Coefficient variation
-* Trimmed Median `*`
-* Trimmed Mean `*`
-* Percentile `**`
+time is applied.
+
+<img src="https://user-images.githubusercontent.com/70198089/236139263-da5579df-8e4d-4ab8-8b36-298ee779ee83.png" width="400"/>
```

Comment:
They've changed this drop down to remove the text. We'll need to include the full explanation of each option in https://github.com/netdata/netdata/blob/master/web/api/queries/README.md#data-grouping (asked Johnny to tell us where it is) and change this screenshot.

Reply:
I am taking note of this.